src/Components/PaginationNav/PaginationDirectionItem.tsx

```diff
@@ -0,0 +1,65 @@
+// Libraries
+import React, {forwardRef, MouseEvent} from 'react'
+import classnames from 'classnames'
+
+// Components
+
+import {Icon} from '../Icon/Base/Icon'
+
+// Styles
+import './Pagination.scss'
+
+// Types
+import {IconFont, StandardFunctionProps, DirectionType} from '../../Types'
+
+export interface PaginationDirectionItemProps extends StandardFunctionProps {
+  /** Caret Left or Caret Right on button */
+  direction: DirectionType
+  onClick?: (e?: MouseEvent<HTMLLIElement>) => void
+}
+
+export type PaginationDirectionItemRef = HTMLLIElement
+export const PaginationDirectionItem = forwardRef<
+  PaginationDirectionItemRef,
+  PaginationDirectionItemProps
+>(
+  (
+    {
+      id,
+      style,
+      testID = 'Pagination-direction-item',
+      className,
+      direction,
+      onClick,
+    },
+    ref
+  ) => {
+    const paginationClassName = classnames(
+      'cf-pagination--direction--button',
+      '',
+      {
+        [`${className}`]: className,
+      }
+    )
+
+    const directionIcon =
+      direction === 'left' ? IconFont['CaretLeft'] : IconFont['CaretRight']
+
+    return (
+      <li
+        className={paginationClassName}
+        data-testid={testID}
+        id={id}
+        style={style}
+        ref={ref}
+        onClick={onClick}
+      >
+        <div className="cf-pagination--direction--icon">
```

Comment:
I'll make this html button for now with the intention that they will eventually be swapped with the library Button. I am thinking we might need to add another type of button to the library that is transparent(tertiary) and has fixed size (I think if I use the library button as is, it will have different width for 1, 11, 111 because it's relational to the content.